.ado/xcconfig/publish_overrides.xcconfig

```diff
@@ -1,2 +1,13 @@
+// For publish builds, only provide line tables for symbolizing crashes
+CLANG_DEBUG_INFORMATION_LEVEL=line-tables-only
+// The following build setting caused build errors, so it is commented out
```

Comment:
Probably worth explaining the actual build error and/or just removing the commented out line altogether? Do we expect to need to add it back at some point?

Reply:
Re Marks' comment here https://github.com/microsoft/fluentui-react-native/pull/710#discussion_r688016727

I wanted to ask what the fallout is by not using it. Looks like it could be a risk so I'm doing some additional testing to make sure we are including the right amount of symbols.

Reply:
I'll continue the thread in the other comment thread